core/CMakeLists.txt

```diff
@@ -61,10 +61,14 @@ add_subdirectory(cont)
 add_subdirectory(dictgen)
 add_subdirectory(foundation)
 add_subdirectory(gui)
-add_subdirectory(imt)
+if(ROOT_ENABLE_PROJECTS AND ROOT_ENABLE_PROJECTS STREQUAL "All")
+  add_subdirectory(imt)
+endif()
```

Comment:
These should go via the new root_add_subdirectory.

Reply:
did changes, however, if necessary, I can exchange all of the add_subdirectory() calls to root_add_subdirectory()